templates/nextflow-ecs-task-definition.j2

```diff
@@ -201,10 +201,6 @@ Resources:
           Image: !Ref BackendContainerImage
           RepositoryCredentials:
             CredentialsParameter: !Sub 'arn:aws:secretsmanager:us-east-1:${AWS::AccountId}:secret:TOWER_DEV_SEQERA_REGISTRY_SECRET'
-        - Name: !Ref CronContainerName
```

Comment:
Should we try unique names as well later? 

!Sub ${BackendContainerName}-secret?